rpc/client/rpc_test.go

```diff
@@ -32,6 +33,19 @@ func GetClients() []client.Client {
 	}
 }
 
+// Make sure status is correct (we connect properly)
```

Comment:
Shouldn't we instead test if CORS is enabled, headers are present and everything works (i.e. HAPPY PATH)?

Reply:
To test Happy path, we need to run another tendermint instance with cors enabled. Is it okey to do in single test?